src/components/button/styles.ts

```diff
@@ -4,7 +4,25 @@ import { colors } from "src/constants";
 export const Container = styled.button`
   display: flex;
   height: 100%;
-  color: ${colors.white};
-  background-color: ${colors.black};
-  border-radius: 12px;
-`;
\ No newline at end of file
+  color: ${colors.black};
+  background: linear-gradient(180deg,#F9DA00 0%,#FD9100 100%);
+  box-shadow: 0px 4px 4px #C8510E;
+  height: 83.9779281616211px;
+  width: 262.4380798339844px;
```

Comment:
What are this width ? Lets use dynamic width like with height, or delete it if you use "width: 100%"